docs/install/install-kn.md

```diff
@@ -34,3 +50,16 @@ Links to either the nightly container image or the latest stable container image
 
 - [Nightly container image](https://gcr.io/knative-nightly/knative.dev/client/cmd/kn)
 - [Latest release](https://gcr.io/knative-releases/knative.dev/client/cmd/kn)
+
+You can run `kn` from container. For example:
+  ```bash
+  docker run --rm -v "$HOME/.kube/config:/root/.kube/config" gcr.io/knative-releases/knative.dev/client/cmd/kn:latest service list
+  ```
+
+## Using `kn` with Tekton
+
+You can also [run kn using Tekton](https://github.com/tektoncd/catalog/tree/master/kn).
+
+## What's next
+Now that you have installed `kn` in your system. To learn more about `kn`, you can read our [document](https://github.com/knative/client/blob/master/docs/cmd/kn.md).
```

Comment:
```suggestion
To learn more about using `kn`, see the [documentation](https://github.com/knative/client/blob/master/docs/cmd/kn.md).
```

Reply:
I'd also add a <!--TODO--> comment here to make sure this gets updated to a link within the docs at some point, rather than the client repo. @rhuss cc'ing you since you might want to update this once there's a `kn` reference section in docs.

Reply:
Thanks for your comments.